stable/linkerd/templates/NOTES.txt

```diff
@@ -0,0 +1,18 @@
+Add ports on your services that have the name {{ .Values.linkerd.portName }}
```

Comment:
By default this statement renders as

> Add ports on your services that have the name service

I feel that's a little vague given no prior knowledge to linkerd. Can we provide an example and make that `service` pop a little?